sql/mozfun/norm/browser_version_info/udf.sql

```diff
@@ -9,7 +9,7 @@ RETURNS STRUCT<
   STRUCT(
     version_string AS version,
     norm.truncate_version(version_string, 'major') AS major_version,
-    norm.truncate_version(version_string, 'minor') AS minor_version,
+    norm.extract_version(version_string, 'minor') AS minor_version,
     norm.extract_version(version_string, 'patch') AS patch_revision,
```

Comment:
After sleeping on it I might retract my suggestions above.  Looking at [Firefox version numbers](https://www.mozilla.org/en-US/firefox/releases/) (presumably the primary use case for this), since they don't prefix single-digit minor version numbers with a zero I guess there is a gotcha with the existing `minor_version` behavior in cases where the minor version hit double-digits (e.g. "91.1.0" and "91.10.0" would both get `minor_version = 91.1`), and maybe it'd be best to not have a partially unreliable `minor_version` behavior (though that only appears to have happened three times so far, with major versions 91, 78, and 68).

In any case, if you do opt to continue with changing `minor_version` to be an integer I'd still suggest also renaming `patch_revision` to `patch_version` for consistency.
```suggestion
    norm.extract_version(version_string, 'minor') AS minor_version,
    norm.extract_version(version_string, 'patch') AS patch_version,
```

Reply:
@sean-rose Thank you so much for the suggestions.  Given the feedback I have gathered so far, I am going to continue with changing `minor_version` as in the existing PR.  If needed, the `version` column is still available for users to truncate when querying the tables or set up lookml.  I will also check with DS to make sure I get their input.  Much appreciated!

Reply:
The column name change for patch_revision has a greater change impact.  I will not include this change with this PR but I will make a note of this.